### PR TITLE
restructure on organisation templates select

### PR DESCRIPTION
**Pull request by squallstar**
will probably conflict with https://github.com/Fliplet/fliplet-widget-form-builder/pull/80 unless we merge this first and update #80 later.

---

img
  
img2

### CODE REVIEW

File: css/builder.css [hunk 1]
```diff
@@ -21,6 +21,24 @@ html {
...
+
+.btn-secondary:hover,
+.btn-secondary:active,
+.btn-secondary:focus,
+.btn-secondary:active:hover,
+.btn-secondary:active:focus {
+  color: #00abd1;
+  background-color: transparent;
+  border-bottom-color: #007994;
+}
```
Comment (tonytlwu) [inline css/builder.css]: Maybe `.btn-secondary` should be added to `fliplet-studio-ui`?

File: js/libs/builder.js [hunk 2]
```diff
@@ -77,13 +77,19 @@ function generateFormDefaults(data) {
     name: '',
     dataSourceId: '',
     templateId: '',
+    previewingTemplate: '',
```
Comment (squallstar) [inline js/libs/builder.js]: what's this? @hcarneiro

File: js/libs/builder.js [hunk 3]
```diff
@@ -703,6 +757,9 @@ var app = new Vue({
         email: data.email,
         type: 'to'
       });
+    },
+    'settings.template': function() {
+      this.editor.setContent(this.settings.description);
```
Comment (squallstar) [inline js/libs/builder.js]: I guess you only need this when it's turned on, right? the first argument in the function should be the value

File: js/libs/builder.js (hunk 2 above)
Comment (hcarneiro) [reply js/libs/builder.js]: @squallstar That's a flag for when the user is previewing the template.

File: css/builder.css [hunk 4]
```diff
@@ -21,6 +21,24 @@ html {
   text-align: center;
 }
 
+.btn-secondary {
+  color: #00abd1;
+  background-color: transparent;
+  border-bottom-color: #00abd1;
+  border-color: #00abd1;
```
Comment (tonytlwu) [inline css/builder.css]: Not sure why `border-bottom-color` and `border-color` are needed

File: css/builder.css [hunk 5]
```diff
@@ -21,6 +21,24 @@ html {
...
+}
+
+.btn-secondary:hover,
+.btn-secondary:active,
+.btn-secondary:focus,
+.btn-secondary:active:hover,
+.btn-secondary:active:focus {
+  color: #00abd1;
+  background-color: transparent;
+  border-bottom-color: #007994;
```
Comment (tonytlwu) [inline css/builder.css]: `border-bottom-color` is the only one that's changed from the above. Are the other 2 property declarations needed?

**Pull request merged**